srcpkgs/dooble/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'dooble'
+pkgname=dooble
+version=2022.06.15
+revision=1
+build_style=qmake
+configure_args="dooble.pro"
+hostmakedepends="qt5-qmake"
+makedepends="qt5-charts-devel qt5-webengine-devel qt5-webchannel-devel qt5-location-devel"
+short_desc="Dooble, the scientific browser. Minimal, cute, and unusually stable"
+maintainer="Eloi Torrents <elotor@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://textbrowser.github.io/dooble/"
+distfiles="https://github.com/textbrowser/dooble/archive/refs/tags/${version}.tar.gz"
+checksum=3af9a06c477f9317cf279389bd66237109fa205fb5526f4df7909250dc2c8e41
+
+do_install() {
+	vbin Dooble
```

Comment:
based on the [AUR package](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dooble):
```suggestion
	vbin Dooble dooble
	vinstall dooble.desktop 644 usr/share/applications
	vinstall Icons/Logo/dooble.png 644 usr/share/pixmaps
	vlicense LICENSE
```
(renamed the bin file so it plays nice with the desktop file)

would be nice to also add the translations, but it seems to assume they're in `$PWD/Translations`. might be something to bring up with the upstream.
```
	vmkdir usr/share/dooble/Translations
	vcopy "Translations/dooble_*.qm" usr/share/dooble/Translations
```

Reply:
Thank you I'll take a look at it and open an issue upstream

Reply:
Patch this file: https://github.com/textbrowser/dooble/blob/94498e29d616de51287e752c57e287dcdccdc578/Source/dooble_application.cc#L81

Reply:
Here is the patch:
```diff
--- dooble_application.cc.orig	2022-09-23 20:15:31.021472658 +0700
+++ dooble_application.cc	2022-09-23 20:16:49.754946070 +0700
@@ -81,9 +81,7 @@
 
       if(path.isEmpty())
 	ok = m_translator->load("dooble_" + QLocale::system().name(),
-				QDir::currentPath() +
-				QDir::separator() +
-				"Translations");
+				"/usr/share/dooble/translations");
       else
 	ok = m_translator->load("dooble_" + QLocale::system().name(), path);
 

```

Reply:
Thank you!!!!!!!

Now only the aarch64 builds give problems...

Reply:
don't forget to install the translations too
```
	vmkdir usr/share/dooble/Translations
	vcopy "Translations/dooble_*.qm" usr/share/dooble/translations
```

Reply:
> Now only the aarch64 builds give problems...

add qt5-host-tools to hostmakedepends